src/main/java/arrays/homework_3/SortingAlgorithms.java

```diff
@@ -1,10 +0,0 @@
-package arrays.homework_3;
-
```

Comment:
Restore this file . You were not allowed to delete it

Reply:
done